app/status/views.py

```diff
@@ -6,4 +7,11 @@
 
 @status.route('/_status')
 def status_no_db():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    try:
+        return jsonify(status="ok", app_version=utils.get_version_label(),
+                       db_version=utils.get_db_version())
+
+    except SQLAlchemyError:
+        current_app.logger.exception('Cannot connect to database.')
+        return jsonify(status="error", message="Database is down"), 500
```

Comment:
I think we could still have the app version number included in the status on database error.